# generated by patch-package 6.4.8 on 2022-05-11 12:17:06
#
# command:
#   npx patch-package @types/react-helmet
#
# declared package:
#   @types/react-helmet: 6.1.0
#
diff --git a/node_modules/@types/react-helmet/index.d.ts b/node_modules/@types/react-helmet/index.d.ts
index 3f4c7dc..2f8b64d 100644
--- a/node_modules/@types/react-helmet/index.d.ts
+++ b/node_modules/@types/react-helmet/index.d.ts
@@ -36,6 +36,7 @@ export interface HelmetTags {
 }
 
 export interface HelmetProps {
+    children?: ReactNode;
     async?: boolean;
     base?: any;
     bodyAttributes?: BodyProps;
